xAPI.md

```diff
@@ -255,6 +255,25 @@ learning that SCORM could not enable.
 
 ## 1.0 Summary of the Experience API
 
+The Experience API (Application Programming Interface) - or xAPI as it is commonly called - is a framework that 
+facilitates the description of learning experiences and their communication and storage in a consistent manner 
+in a digital environment. As an application programming interface, the xAPI does not set out how the framework  
+should be implemented. Rather, it sets out the essential components of that framework and how they relate to each other. The purpose of the xAPI and the goals of this document are:
+
+* To maximize interoperability and reliability of computer programs which create, gather and store information  
+about learning experiences.
+* To provide a guide to computer programmers who want to build applications that conform to and implement this 
+framework.
+* To provide users of these programs a description of features and capabilities that they can expect from them.
+* To provide the ADL that certifies conformance to this framework with criteria against which certification can 
```

Comment:
Wondering if we should be mentioning ADL specifically here or whether we should keep ADL references to the ' ADL's Role in the Experience API' section. I don't feel strongly about this either way.